Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -247,10 +253,15 @@ public class Fuzzer {
     ///
     /// This will import each program in the given array into this fuzzer while potentially discarding
     /// some percentage of the programs if dropout is enabled.
-    public func importCorpus(_ corpus: [Program], withDropout applyDropout: Bool = false) {
+    public func importCorpus(_ corpus: [Program], withDropout applyDropout: Bool = false, logStatus: Bool = false) {
```

Comment:
I don't think this function has any other users currently, so I think we can just always log progress without the need for the additional parameter